runtime/doc/treesitter.txt

```diff
@@ -385,14 +385,17 @@ The following directives are built in:
             (#gsub! @_node ".*%.(.*)" "%1")
 <
     `trim!`                                          *treesitter-directive-trim!*
-        Trim blank lines from the end of the node. This will set a new
-        `metadata[capture_id].range`.
+        Trim whitespace from the node. This will set a new
+        `metadata[capture_id].range`. Trims blank lines from the end of the node
+        by default (for backwards compatibility). Trims all whitespace from both
+        sides of the node if the optional parameter "1" is given.
```

Comment:
I was actually envisioning this to work like the `#offset!` directive, where you can control each of the four(!) sides individually.

Reply:
Sure gimme a sec- I assume startline startcols endline endcols?